packages/client/src/offline/selectors.ts

```diff
@@ -33,9 +34,12 @@ export function isOfflineDataLoaded(
     state.forms.userForm &&
     state.languages &&
     state.assets &&
-    state.templates
+    state.templates &&
+    state.config
 
-  return Boolean(hasAllRequiredData)
+  const isOfflineDataLoaded = Boolean(hasAllRequiredData)
+  if (isOfflineDataLoaded) merge(window.config, state.config)
```

Comment:
Would there be another way of doing this? Mutating the global `window.config` inside a call to a previously pure function feels a bit misleading from code reader point of view

Reply:
I think it is OK.  By merging state.config into window.config, the props loaded from the config microservice become globally accessible and then there is no need to refactor other parts of the code to retrieve config from redux state